### DISCUSSION TITLE
arc<hub> or arc<auth> then constract hub?

### DISCUSSION THREAD

**Discussion by mohad12211** [Byron/google-apis-rs#439] [Q&A] [answered]
for shared state, should I wrap the `hub` in an arc or only the `auth` and constract a `hub` each time I want to make a request?

**Comment by Byron** [comment, marked as answer]
I know it's a none-answer, but use whatever works for you. Rust makes sure it's safe. If 'one-hub-per-call' is too slow, I am sure there are other ways to share it as well.

**Reply by mohad12211** [reply]
I see, thank you!
